components/automate-cli/pkg/verifyserver/server/api/v1/systemresources.go

```diff
@@ -0,0 +1,24 @@
+package v1
+
+import (
+	"net/http"
+
+	"github.com/chef/automate/components/automate-cli/pkg/verifyserver/response"
+	"github.com/gofiber/fiber"
+)
+
+func (h *Handler) GetSystemResource(c *fiber.Ctx) {
+
+	nodeType := c.Query("node_type", "")
+	deploymentState := c.Query("deployment_state", "")
+	service, err := h.SystemResourceService.GetSystemResourcesForDeployment(nodeType, deploymentState)
```

Comment:
Add validation check for nodeType, deploymentState before passing it to the service
PS: Check for bad request here itself

Reply:
Done